andela/ah-orcas-frontend
# #162706980 Fix social login bug

#### What does this PR do?
```This PR redesigns the social buttons and fixes a bug in social login when a user clicks the button twice ```
#### Description of Task to be completed?
```Redesign the social buttons to look better```
```Fix bug in social login```
#### How should this be manually tested?
- clone this repository
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
```cd ah-orcas-frontend```
- Checkout to the appropriate branch
```git checkout bg-fix-social-login-162706980```
Install the required dependencies and start the server
- npm install
 - npm start
```You should be able to see the new design of the social buttons on the login page.```

#### What are the relevant pivotal tracker stories?
[#162706980](https://www.pivotaltracker.com/story/show/162706980)
#### Screenshots 
- Add screen short if available
![image](https://user-images.githubusercontent.com/25586421/50267898-e1d52500-0439-11e9-9689-3bd23b9cfb0a.png)


#### Checklist:
